test/integration/model.test.js

```diff
@@ -277,7 +277,7 @@ describe(Support.getTestDialectTeaser('Model'), function() {
         bCol: { type: Sequelize.STRING, unique: 'a_and_b' }
       });
 
-      return User.sync({ force: true, logging: _.after(2, _.once(function(sql) {
```

Comment:
What is the warning logged here - I think we should rather remove it than working around it. This breaks tests for other dialects